docker-compose.env.sample

```diff
@@ -2,4 +2,11 @@ DATABASE_URL=postgres://postgres@db:5432/DataSubmissionServiceApi_development?te
 REDIS_URL=redis://redis:6379/
 AWS_ACCESS_KEY_ID=
 AWS_SECRET_ACCESS_KEY=
-AWS_S3_REGION=
+AWS_S3_REGION=eu-west-2
+AWS_S3_BUCKET=dss-infrastructure-development-storage
+GOOGLE_CLIENT_ID=
+GOOGLE_CLIENT_SECRET=
+ADMIN_EMAILS=
+AUTH0_DOMAIN=
```

Comment:
I didn't think the API used auth0 only Google?

Reply:
The app speaks to Auth0 to create a user there when a new user is added in the application. Same when a user is deleted – it removes the user from Auth0.

Reply:
Got it! Thanks!